### PR TITLE
Add helper methods to AzureClient and AzureSilo for easier programmatic config

### PR DESCRIPTION
**Pull request by sergeybykov**
Add AzureClient.DefaultAzureConfig() and AzureSilo.DefaultAzureConfig() helper methods.

Usage examples are in https://github.com/sergeybykov/orleans/blob/sample/Samples/AzureWebSample/WebRole/Default.aspx.cs#L16 and https://github.com/sergeybykov/orleans/blob/sample/Samples/AzureWebSample/OrleansAzureSilos/WorkerRole.cs#L49.

### CODE REVIEW

File: src/OrleansAzureUtils/Hosting/AzureClient.cs [hunk 1]
```diff
@@ -123,12 +140,12 @@ private static void InitializeImpl_FromFile(FileInfo configFile)
             InitializeImpl_FromConfig(config);
         }
 
-        private static string GetDeploymentId()
+        internal static string GetDeploymentId()
         {
             return GrainClient.TestOnlyNoConnect ? "FakeDeploymentId" : serviceRuntimeWrapper.DeploymentId;
```
Comment (veikkoeeva) [inline src/OrleansAzureUtils/Hosting/AzureClient.cs]: Out of curiosity, would there be a need to use "FakeDeploymentId" elsewhere? E.g. in some comparison operator?

Comment (sergeybykov) [inline src/OrleansAzureUtils/Hosting/AzureClient.cs]: I don't know. I don't even see it being used in tests. Probably a leftover from the past. But this PR isn't concerned with it.

**Comment by sergeybykov** [comment]
@dotnet-bot test this please

**Comment by sergeybykov** [comment]
I renamed "DefaultConfig" to "DefaultConfiguration".

**Pull request merged**